### PR TITLE
Polyglot string function

### PR DESCRIPTION
**Pull request by franalderete7**

### CODE REVIEW

**Comment by franalderete7** [comment]
Did not find many strings like these. Only the ones on this branch.

File: components/Prompts.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import api, { formatSearch, cdn } from '../utils/api';
 import UploadPhoto from './UploadPhoto';
 
 import { useAuth } from '../contexts/auth.js';
+import polyglot from '../locales/base';
```
Comment (closerearth) [inline components/Prompts.js:8]: Instead of importing it directly, please wrap the `__` function from helpers.

**Comment by closerearth** [comment]
Usability looks good - but let's change the structure a bit.
Let's remove base.js and instead use polyglot within the `__` helper.js function instead of the simple object lookup that was there.

File: pages/search/[keyword].js [hunk 2]
```diff
@@ -4,59 +4,64 @@ import Link from 'next/link';
...
-                      <a role="button">
-                        <span className="title">
-                          {article.title}
-                        </span>
-                        { article.summary &&
+const Search = ({ articles, error, keyword, tags }) => {
+  
+  var polyglot = new Polyglot();
+  
+  console.log(error, articles) || (
```
Comment (closerearth) [inline pages/search/[keyword].js:14]: 👀

File: components/Prompts.js [hunk 3]
```diff
@@ -26,8 +28,8 @@ const Prompts = () => {
         <div className="main-content p-3 justify-between flex flex-col relative text-center shadow-sm">
           <div className="p-2">
             { image ?
-              <p>{ polyglot.t('image_true', { name: user.screenname }) }</p>:
-              <p>{ polyglot.t('image_false', { name: user.screename })  }</p>
+              <p>{ __(polyglot.t('image_true', { name: user.screenname })) }</p>:
```
Comment (closerearth) [inline components/Prompts.js:31]: No - what I want is `<p>{ __('image_true', { name: user.screenname }) }</p>`
And then in the __ function, use polyglot.

**Pull request closed without merging**